src/utils/css-utils.js

```diff
@@ -5,6 +5,7 @@ const sizes = {
     desktop: 1200,
     tablet: 768,
     phone: 599,
+    verySmall: 280,
```

Comment:
Oh, yes! Media-query should use `min-width` and we should add rule for `media.phone` as `max-width: 768`. 
@ifedyukin Please, fix this.